src/main/clojure/jarl/eval.clj

```diff
@@ -143,7 +143,7 @@
   (log/tracef "NotStmt - <%s> == <%s>" a-pos b-pos)
```

Comment:
This log line seems wrong, if not changed now.

Reply:
I could have sworn I've already fixed this. Must have been a similar mistake somewhere else 😄 .

Reply:
👍😄